api/filters.go

```diff
@@ -2,6 +2,7 @@ package api
 
 import (
 	"context"
+	"github.com/go-chi/chi/v5"
```

Comment:
It's generally preferred to have some sorting with the imports along the lines of, - keeping standard library, ONS packages and third party packages in 2 separate groups and sorted alphabetically
```suggestion
import (
        "context"
        "net/http"
        "strconv"
        
	"github.com/go-chi/chi/v5"
```